includes/class-demo-poem.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Demo poem class.
+ *
+ * @link       https://nataliesmith.dev
+ * @since      1.0.0
+ * @package    Poetwp\Includes
+ */
+
```

Comment:
This file isn't namespaced, which means the `Demo_Poem` class is sitting in the global space.

Consider adding a namespace to this file, and then update the files that reference it with a `use` import statement.